ipython/kinetics_library_to_training_tools.py

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env python
```

Comment:
Since this file does not have a `main()` function that would get executed if you were to run this as a script, should we move this file to `rmgpy/tools/` where code related to this is stored?

Reply:
I considered that, but also wanted to keep it close to the ipython notebook. I can move it if you think it would be better.